README.md

```diff
@@ -765,6 +767,23 @@ const pdfBytes = await pdfDoc.save()
 //   • Rendered in an <iframe>
 ```
 
+### Extract Attachments
+
+If you load a PDF that has `cars.csv` as an attachment, you can use the
+following to extract the attachments:
+
+<!-- prettier-ignore -->
+```js
+const pdfDoc = await PDFDocument.load(...)
+const attachments = pdfDoc.getAttachments()
+const csv = attachments.find(({ name }) => name === 'cars.csv')
+fs.writeFileSync(csv.name, csv.data)
+```
+
+> NOTE: If you are building a pdf file with this library, any attachments you've
```

Comment:
I see no reason why not? This might be confusing for users...

Reply:
Where are the attachments being stored before they are serialized? We can probably include that location when returning the attachments, no?

Reply:
I agree that would be better. It's been a while since I've looked at this code, but I vaguely remember that being difficult. I see a comment on `PDFEmbeddedFile.embed` that says something to the effect of it not being embedded until `save` is called. I also don't see how to get this information out of `PDFEmbeddedFile`.

I won't be able to spend a lot of time on this soon, but I'm planning to come back at some point and give it another shot to see if what you're saying is possible.